.prettierignore

```diff
@@ -0,0 +1,13 @@
+package*.json
+
+*.md
+
+**/task.json
+vss-extension.json
+
+# Temporarily ignore the following files to reduce the number of changes.
+# Will remove in a follow-up PR.
```

Comment:
Could you file a quick issue to track this? If it were me, I'd forget to file the follow-up PR  unless there's an issue to remind me :D